packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -95,11 +95,11 @@ function makeLeaf<T extends FlexLeafNodeSchema>(
 }
 
 // Leaf schema shared between all SchemaFactory instances.
-const stringSchema = makeLeaf(leaf.string);
-const numberSchema = makeLeaf(leaf.number);
-const booleanSchema = makeLeaf(leaf.boolean);
-const nullSchema = makeLeaf(leaf.null);
-const handleSchema = makeLeaf(leaf.handle);
+export const stringSchema = makeLeaf(leaf.string);
```

Comment:
@CraigMacomber I needed `nullSchema` in particular in `toMapTree`. The others I exported for unit test convenience. Let me know if you would prefer to avoid this / have alternative suggestions.

Reply:
These are all available on schema factory instances as members, so you can just new up a schema factory to access them.

If people want access to them without a schema factory instance, personally I'd go for adding them as static members to SchemaFactory. Personally I think that make more sense than as instance members anyway, but thats not how the spec did it,

Reply:
Hmm. New-ing a schema factory inline to get the definition of null seems odd, but it will do the trick. I'll just leave a comment that if we ever move to exposing them as statics, to use those.

Reply:
Actually doing this in `toMapTree` actually causes problems due to the import cycle between `toMapTree` and `schemaFactory` (via `proxies`).

Extracting the leaf schemas into their own module seems to have resolved the issue, and has reduced the cycle count detected by depcruiser. Open to alternatives, though.